spring-integration-amqp/src/main/java/org/springframework/integration/amqp/channel/PollableAmqpChannel.java

```diff
@@ -40,8 +43,8 @@
  * @author Gary Russell
  * @since 2.1
  */
-public class PollableAmqpChannel extends AbstractAmqpChannel implements PollableChannel,
-		PollableChannelManagement {
+public class PollableAmqpChannel extends AbstractAmqpChannel
+		implements PollableChannel, PollableChannelManagement, ExecutorChannelInterceptorAware {
```

Comment:
Unfortunately, this name now looks wrong here.

We can't change `ExecutorChannelInterceptor` (which is also technically wrong), but maybe we could call this `HandlerInvokerInterceptorAware` WDYT?